app/services/placement_export_csv_service.rb

```diff
@@ -0,0 +1,39 @@
+require "csv"
+
+class PlacementExportCsvService
+  attr_reader :placements
+
+  def initialize(casa_org_id:)
+    @casa_org = CasaOrg.find(casa_org_id)
+  end
+
+  def perform
+    placements = fetch_placements
+
+    CSV.generate(headers: true) do |csv|
+      csv << full_data.keys.map(&:to_s).map(&:titleize)
+      if placements.present?
```

Comment:
You can actually leave off `if placements.present?` since `placements` will be an array, and it just won't loop if it's empty.